libs/wingc/src/jsify.rs

```diff
@@ -53,6 +54,8 @@ pub struct JSifier<'a> {
 	pub types: &'a mut Types,
 	/// Store the output files here.
 	pub output_files: RefCell<Files>,
+	/// Stored struct schemas that are referenced in the code.
+	pub referenced_struct_schemas: RefCell<BTreeMap<String, CodeMaker>>,
```

Comment:
I'm curious how this works for multi-file programs. I have a sense either the file names (or some associated counter) needs to be part of the keys, or we might need to store one of these maps per AST.

(I'm ok with leaving this for a follow up issue)

Reply:
Ah cool - LGTM